styles/nav.scss

```diff
@@ -81,6 +81,7 @@ $daptiv-subnav-tab-h-pad: 14px;
         padding-right: $standard-whitespace-unit;
         text-decoration: none;
         text-overflow: ellipsis;
+        overflow: hidden;
```

Comment:
Good point. I have created a separate branch to add SCSS linting so we can follow and know about the standards.